01week/rockPaperScissors.js

```diff
@@ -8,11 +25,26 @@ const rl = readline.createInterface({
 });
 
 
-function rockPaperScissors(hand1, hand2) {
-
-  // Write code here
-
+// MY CODE //
+function rockPaperScissors(hand1, hand2){
+   if (hand1 === 'rock' && hand2 === 'rock') {
+     console.log("It's a tie!");
+   } else if (hand1 === 'paper' && hand2 === 'paper') {
+     console.log("It's a tie!");
+   } else if (hand1 === 'scissors' && hand2 === 'scissors') {
+     console.log("It's a tie!");
+   } else if (hand1 === 'rock' && hand2 === 'rock') {
+     console.log("It's a tie!");
```

Comment:
This function should make a return statement instead of a console.log(). Console.log is a great way to PRINT comments on your terminal but I does not return a value.